package.json

```diff
@@ -24,7 +24,6 @@
     "knex": "^0.14.6",
     "nodemailer": "^4.6.4",
     "nodemailer-express-handlebars": "^3.0.0",
-    "objection": "^1.1.9",
```

Comment:
![image](https://user-images.githubusercontent.com/71928/43622768-7b648706-96a3-11e8-8b31-478dd5a95fbf.png)

Reply:
Leave now and never come back!